Sources/TuistKit/Generator/ProjectFileElements.swift

```diff
@@ -123,7 +123,10 @@ class ProjectFileElements {
         }
 
         // Support files
-        files.insert(target.infoPlist)
+        if let infoPlist = target.infoPlist {
```

Comment:
Is Xcode happy about having a framework without a `Info.plist`?

Reply:
Surprisingly yes, that seems to be the case for the Swift PM package description target:

<img width="814" alt="tuist-packageDescription" src="https://user-images.githubusercontent.com/11914919/54295328-37017c80-45cc-11e9-951e-202dbe6b8138.png">